Change type of Peer.EndPoint to DnsEndPoint (#166)
planetarium/libplanet: CHANGES.md

 -  Since we decided to depend on TURN ([RFC 5766]) and STUN ([RFC 5389]) to
    work around NAT so that `Peer`'s endpoints don't have to be multiple,
    `Peer.Urls` was renamed to `Peer.EndPoint` and its type also was changed
    from `IImmutableList<Uri>` to `DnsEndPoint`.

>     [[#120], [#123] by Yang Chun Ung, [#126], [#127], [#165]]

Reviewer: It would be good if it mentions this PR #166 as well.

 -  `Swarm` became to ignore tip blocks of the same height (`Index`) that it
    already has and deal with only longer (higher) blocks.
 -  Fixed a bug that occured when `Swarm` was handling multiple responses at the
    same time.

>  -  Fixed a bug that hangs when calling the `Swarm` constructor in certain

Reviewer: ```suggestion
 -  Fixed a bug that the `Swarm` constructor had hanged in certain runtimes
```

>     runtimes. (e.g. Unity3d)

Reviewer: ```suggestion
    like Unity engine.
```

Unity seems to recently have changed their name from <q>Unity3D</q> to simply <q>Unity</q>.
